Cargo.toml

```
@@ -16,8 +16,10 @@ members = [
     "crates/rpc",
     "crates/runtime",
     "crates/storage",
-    "testing/ef-tests",
-]
+    "testing/beacon-api",
+    "testing/beacon-api"
+,
+    "testing/ef-tests"]
```

```suggestion
    "testing/beacon-api",
    "testing/ef-tests"
]
```

Looks like a woopsy was done here